staging/src/k8s.io/component-base/metrics/opts.go

```diff
@@ -53,6 +53,18 @@ const (
 	STABLE StabilityLevel = "STABLE"
 )
 
+// amend validates stability level and takes 'ALPHA' in case of empty.
+func (sl *StabilityLevel) amend() {
+	switch *sl {
+	case ALPHA:
```

Comment:
I feel like this reads more cleanly:

```
	switch *sl {
	case "":
		*sl = ALPHA
	default:
		// no-op, since we have a StabilityLevel already
	}
```